### ISSUE TITLE
Warn and info log levels have different outputs

### ISSUE THREAD

**Issue by petosorus** [rustwasm/wasm-pack#716]
## 🐛 Bug description
Warn level and info levels have different behaviors. Most notably, the warn level has an emoji first, and the log level second.

![image](https://user-images.githubusercontent.com/4728156/65515813-28334500-dee0-11e9-86dc-d8b934a8b43c.png)

I took a look at the code, warn and error are similar, but info is different.
```/// Add an informational message.
    pub fn info(&self, message: &str) {
        let info = format!("{}: {}", style("[INFO]").bold().dim(), message,);
        self.message(&info);
    }

    /// Add a warning message.
    pub fn warn(&self, message: &str) {
        let warn = format!(
            "{} {}: {}",
            emoji::WARN,
            style("[WARN]").bold().dim(),
            message
        );
        self.message(&warn);
    }

    /// Add an error message.
    pub fn error(&self, message: &str) {
        let err = format!(
            "{} {}: {}",
            emoji::ERROR,
            style("[ERR]").bold().dim(),
            message
        );
        self.message(&err);
    }
```

#### 🤔 Expected Behavior
I would expect the log level to always be the first element of the log. I don't mind the emoji, I just find it in the wrong place.

#### 👟 Steps to reproduce
I ran the example from this page https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_wasm

When running `wasm-pack build`, I see the incriminated logs.

#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version: 0.8.1
rustc version: 1.37.0

I would be happy to take a stab at doing a PR for this :)

**Comment by drager** [comment]
Makes sense! Thanks for reporting!

**Comment by ashleygwilliams** [comment]
@drager you marked this as PR attached- did this get solved? let me know!

**Comment by drager** [comment]
@ashleygwilliams There is a PR that hasn't been merged yet: https://github.com/rustwasm/wasm-pack/pull/723

**Comment by drager** [comment]
Fixed in #723
